### ISSUE TITLE
Bug? Column,Row alignment only trigger with explicit height/width

### ISSUE THREAD

**Issue by hololeo** [flet-dev/flet#315]
**This does not work:** The following code does not put the red box at the end of the column:
```python
import flet
from flet import Page, Column, Container

def makeCol (page):
    col = Column (
        alignment = "end",
    )
    box = Container (
        width = 100,
        height=  100,
        bgcolor = "red"
    )
    col.controls.append (box)
    return col

def main(page: Page):
    widget = makeCol (page)    
    page.update()

flet.app(target=main, assets_dir="assets")
```
**This workaround works but no bueno:** Column alignment 'end' or 'center' only appear to work if column has an explicit height defined. 
```python

import flet
from flet import Page, Column, Container

def makeCol (page):
    col = Column (
        alignment = "end",
        height = page.window_height
    )
    box = Container (
        width = 100,
        height=  100,
        bgcolor = "red"
    )
    col.controls.append (box)
    return col

def main(page: Page):
    widget = makeCol (page)    
    page.update()

flet.app(target=main, assets_dir="assets")
```

However this is undesirable since it will require hard coding everything up the parent tree. It would be much nicer for Column, Row, WhateverLayoutContainer to get its width/height autofilled as things change. 

I believe Flutter has a solution for this called IntrinsicHeight & IntrinsicWidth. See these videos

https://www.youtube.com/watch?v=8IhtLdtXAzk
https://www.youtube.com/watch?v=ce07f8ClcMQ

Basically it will be very helpful for Flet to autofill the properties that affect a widget during run time. 
To save on websocket chatter, perhaps the dev can have an api like control.getIntrinsics() that will have the dart client return and provide the heigh/width of the column/row whatever

https://api.flutter.dev/flutter/widgets/IntrinsicHeight-class.html

but note its caution:

_This class is relatively expensive, because it adds a speculative layout pass before the final layout phase. Avoid using it where possible. In the worst case, this widget can result in a layout that is O(N²) in the depth of the tree._

**Comment by FeodorFitsner** [comment]
How about this?

```python
import flet
from flet import Column, Container, Page


def makeCol(page):
    col = Column(alignment="end", expand=True)
    box = Container(width=100, height=100, bgcolor="red")
    col.controls.append(box)
    return col


def main(page: Page):
    widget = makeCol(page)
    page.add(widget)

flet.app(target=main)
```

**Comment by hololeo** [comment]
i had tried expand but wasnt working, but with your code it works. i will investigate further to break your example ;)

**Comment by hololeo** [comment]
you can close this if you want. i think it was user error (ie me). 
but there are intrinsic layout properties that would be great if flet could query dart to provide so the developer has better properties to work against as a default when they create containers

**Comment by FeodorFitsner** [comment]
Closing.
